### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by superman00111** [Chia-Network/chia-blockchain#7850] [Support]
After updating version 1.2.3, Chia wallet cannot load 400 harvester nodes at the same time.When I loaded 400 harvesters, Chia software did not respond.After waiting for several days, it's still abnormal.

**Comment by keliew** [comment]
what's the error message in `debug.log`? do you have multiple wallet keys?

**Comment by emlowe** [comment]
More details are required for this bug report - please submit logs or log snippets with errors/warnings. 
Do you mean to say you have 400 harvester nodes connected to 1 farmer node?

**Comment by superman00111** [comment]
yes，I have 400 harvester nodes. farmer node has no errors
It may be because there are too many files，loading the plot file is slow 。

**Comment by keliew** [comment]
how many plots in each harvester?

what does log say about plots loading times? or anything...
